inspirehep/modules/records/serializers/impactgraph_serializer.py

```diff
@@ -90,14 +90,13 @@ def serialize(self, pid, record, links_factory=None):
         ]
 
         if reference_recids:
-            mget_body = {
-                "ids": reference_recids
-            }
+            query = IQ(' OR '.join('recid:' + str(ref)
```

Comment:
Thanks. This `IQ` is just a small wrapper so that we can pass either Invenio or Spires syntax, but in the end it uses `elasticsearch_dsl` - https://github.com/inspirehep/inspire-next/blob/f9ca62ed9bb8fc7b5d35d5e0517eab1b5e8c8df1/inspirehep/modules/search/query_factory.py#L68

Is that what you mean?